### ISSUE TITLE
Speedport Pro Plus - Status 307

### ISSUE THREAD

**Issue by Maximapple** [janh/go-dsl#1]
Unable to read DSL Stats of the Speedport Pro Plus running Firmware 120141.5.0.013.0

```
PS C:\Users\Maxim\Downloads\go-dsl-master> ./dsl -d speedport speedport.ip

Connecting… failed: request for /html/login/index.html failed with status 307
PS C:\Users\Maxim\Downloads\go-dsl-master> ./dsl -d speedport 10.10.1.1

Connecting… failed: request for /html/login/index.html failed with status 307
```

**Comment by janh** [comment]
Speedport Pro is supported by the Sagemcom client, so please try the following command instead:
```
./dsl -d sagemcom speedport.ip
```

If that doesn't fix the problem, please reply and I'll reopen the issue.

**Comment by Maximapple** [comment]
It also doesn't work.

```
PS C:\Users\Maxim\Downloads\go-dsl-master> ./dsl -d sagemcom speedport.ip

Connecting… password required
Password:
Authenticating… failed: json: cannot unmarshal string into Go struct field xmoLoginReplyParameters.id of type uint32
PS C:\Users\Maxim\Downloads\go-dsl-master>
```

**Comment by janh** [comment]
Make sure to use the latest version. This should already be fixed by commit 641e0cc1438b97c2dc8d8b338bd22e525f0ba4c6.

**Comment by Maximapple** [comment]
I had to rebuild the exe file, but it is working properly now. Thanks!
